platform-demo-module/src/test/java/com/softicar/platform/demo/module/invoice/transients/sorting/DemoInvoicesTransientFieldFilteringTest.java

```diff
@@ -0,0 +1,53 @@
+package com.softicar.platform.demo.module.invoice.transients.sorting;
```

Comment:
Should this really reside in the `sorting` package?

Reply:
indeed, moved both test classes up